### PR TITLE
Shorten test run time

### PR DESCRIPTION
**Pull request by ahmetmircik**

### CODE REVIEW

File: hazelcast/src/test/java/com/hazelcast/cache/eviction/CacheExpirationTest.java [hunk 1]
```diff
@@ -77,10 +85,10 @@
     @Parameterized.Parameter(0)
     public boolean useSyncBackups;
 
-    private final Duration FIVE_SECONDS = new Duration(TimeUnit.SECONDS, 5);
+    private final Duration THREE_SECONDS = new Duration(TimeUnit.SECONDS, 3);
```
Comment (mmedenjak) [inline hazelcast/src/test/java/com/hazelcast/cache/eviction/CacheExpirationTest.java]: Is this going to make the tests more timing sensitive (flaky)?

File: hazelcast/src/test/java/com/hazelcast/cache/eviction/CacheExpirationTest.java [hunk 2]
```diff
@@ -66,6 +68,12 @@
     @Rule
     public final OverridePropertyRule overrideTaskSecondsRule = set(PROP_TASK_PERIOD_SECONDS, "1");
 
+    @Rule
+    public final OverridePropertyRule overrideCleanupPercentage = set(PROP_CLEANUP_PERCENTAGE, "100");
+
+    @Rule
+    public final OverridePropertyRule overrideCleanupOperationCount = set(PROP_CLEANUP_OPERATION_COUNT, "1000");
+
```
Comment (mmedenjak) [inline hazelcast/src/test/java/com/hazelcast/cache/eviction/CacheExpirationTest.java]: While you're at it, you can use `smallInstanceConfig()` in this and the other test as the partition/event  count

File: hazelcast/src/test/java/com/hazelcast/cache/eviction/CacheExpirationTest.java (hunk 1 above)
Comment (ahmetmircik) [reply hazelcast/src/test/java/com/hazelcast/cache/eviction/CacheExpirationTest.java]: lets see.

File: hazelcast/src/test/java/com/hazelcast/cache/eviction/CacheExpirationTest.java (hunk 2 above)
Comment (ahmetmircik) [reply hazelcast/src/test/java/com/hazelcast/cache/eviction/CacheExpirationTest.java]: done.

**Pull request merged**